plugins/aws/documentDB/docdbCertificateRotated.js

```diff
@@ -0,0 +1,76 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'DocumentDB Cluster Instance Certificate Rotation',
+    category: 'DocumentDB',
+    domain: 'Databases',
+    severity: 'High',
+    description: 'Ensure that AWS DocumentDB cluster instance certificates are rotated.',
+    more_info: 'Enabling the AWS DocumentDB cluster certificate rotation ensures that your cluster\'s TLS certificates are automatically rotated to maintain security and compliance standards. This feature helps in seamlessly updating certificates without downtime, ensuring continuous protection for data in transit within the DocumentDB cluster.',
+    recommended_action: 'Modify DocumentDB cluster instance and rotate the old server certificate.',
+    link: 'https://docs.aws.amazon.com/documentdb/latest/developerguide/ca_cert_rotation.html',
+    apis: ['RDS:describeDBInstances'],
```

Comment:
why are we not using the DocDB:describeDBClusters call?

Reply:
We need to list the docdb instances here. And DocDb:describeDBInstances and rds:describeDBInstances, lists same data

Reply:
We need to list the docdb instances here. And DocDb:describeDBInstances and rds:describeDBInstances, lists same data